src/assisted_test_infra/test_infra/utils/oc_utils.py

```diff
@@ -19,13 +19,21 @@ def extend_parser_with_oc_arguments(parser):
         type=str,
         default="http",
     )
+    # to be removed
     parser.add_argument(
         "-ocs",
         "--oc-server",
         help="Server for oc login, required if --oc-token is provided",
         type=str,
         default="https://api.ocp.prod.psi.redhat.com:6443",
     )
+    parser.add_argument(
+        "-odf",
```

Comment:
?
I don't think ``ocs-`` above refers to OCS, it's just an acronym for oc server

Reply:
For sure, thanks